src/convnets/convnext.jl

```diff
@@ -58,13 +58,13 @@ function convnext(depths, planes; inchannels = 3, drop_path_rate = 0., λ = 1f-6
     cur += depths[i]
   end
 
-  layers = collect(Iterators.flatten(Iterators.flatten((zip(downsample_layers, stages)))))
-
+  backbone = collect(Iterators.flatten(Iterators.flatten((zip(downsample_layers, stages)))))
```

Comment:
```suggestion
  backbone = collect(Iterators.flatten(Iterators.flatten(zip(downsample_layers, stages))))
```
If my eyes don't trick me